website/common/locales/en/groups.json

```diff
@@ -136,7 +137,6 @@
   "gemGiftsAreOptional": "Please note that Habitica will never require you to gift gems to other players. Begging people for gems is a <strong>violation of the Community Guidelines</strong>, and all such instances should be reported to <%= hrefTechAssistanceEmail %>.",
   "battleWithFriends": "Battle Monsters With Friends",
   "startAParty": "Start a Party",
-  "possessiveParty": "<%= name %>'s Party",
```

Comment:
I think this was removed by mistake

Reply:
I just saw those in my editor, it shows duplicate keys in the files, there still should be one there